chart/kubeapps/values.yaml

```diff
@@ -292,8 +292,8 @@ authProxy:
   # Image used for the proxy
   image:
     registry: docker.io
-    repository: keycloak/keycloak-gatekeeper
-    tag: 4.6.0.Final
+    repository: bitnami/keycloak-gatekeeper
+    tag: 2.3.0-r0
```

Comment:
why the version difference?

Reply:
Because for the images they were using this versioning (I guess it's the version of Keycloak itself):
https://github.com/jboss-dockerfiles/keycloak/releases

But the latest release for keycloak-gatekeeper is that one: 
https://github.com/keycloak/keycloak-gatekeeper/releases

Reply:
Weird, thanks!